mathesar_ui/src/pages/schema/TableCard.svelte

```diff
@@ -195,7 +211,7 @@
     flex: 1 1 auto;
     overflow: hidden;
     font-size: var(--text-size-large);
-    height: var(--menu-trigger-size);
+    height: calc(var(--menu-trigger-size) + 1rem);
```

Comment:
I assume this calculation is done inorder to avoid the menu button focus state from overlapping the 'Find record' button.

Instead of this calculation, I'd recommend to increase the value of `--menu-trigger-size` variable as it is meant to represent the exact size of the menu, and use calc to decrease the height of the menu button based on your needs.

Reply:
done